src/main.rs

```diff
@@ -148,6 +148,14 @@ enum Commands {
     },
     /// List probes
     List {},
+    /// Enable 3.3V output
+    Enable3v3 {},
+    /// Disable 3.3V output
+    Disable3v3 {},
+    /// Enable 5V output
+    Enable5v {},
+    /// Disable 5V output
+    Disable5v {},
```

Comment:
Better make them grouped into a `SetPower(commands::SetPower)` sub-command.

Reply:
Thank you for review.
I grouped then into sub-command.